python/tests/spark/sql/codegen/test_mlflow_registry.py

```diff
@@ -11,6 +11,7 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
```

Comment:
Shall we revert this change, given that this is not a functional change

Reply:
Yes. This commit was used to trigger CI.